### ISSUE TITLE
Shell (or command) fails in Ubuntu 15.04

### ISSUE THREAD

**Issue by figure002** [ansible/ansible#15345]
##### ISSUE TYPE
- Bug Report

This is the same bug as reported in issue #14444, which was incorrectly closed (they thought it was Ansible Tower specific, but it's not). And issue #15328 is very similar to this one.
##### ANSIBLE VERSION

```
ansible 2.0.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = /usr/share/ansible
```
##### OS / ENVIRONMENT

Ubuntu 15.04
And Ubuntu 14.04.4 LTS is the OS I'm managing.
##### SUMMARY

Playbooks fail at the `shell` or `command` modules. Ansible crashes with:

```
AttributeError: 'tuple' object has no attribute 'items'
```
##### STEPS TO REPRODUCE

Run any playbook that has a `shell` or `command` task:

```
- name: Initialize the alias database
  command: /usr/bin/newaliases
```
##### ACTUAL RESULTS

This is the error message I get:

```
RUNNING HANDLER [postfix : Initialize the alias database] **********************
fatal: [jedi]: FAILED! => {"changed": false, "failed": true, "module_stderr": "", "module_stdout": "\r\nTraceback (most recent call last):\r\n  File \"/home/vagrant/.ansible/tmp/ansible-tmp-1460189561.45-24282464722889/command\", line 2402
, in <module>\r\n    main()\r\n  File \"/home/vagrant/.ansible/tmp/ansible-tmp-1460189561.45-24282464722889/command\", line 118, in main\r\n    module = CommandModule(argument_spec=dict())\r\n  File \"/home/vagrant/.ansible/tmp/ansible-tmp
-1460189561.45-24282464722889/command\", line 746, in __init__\r\n    self._check_arguments(check_invalid_arguments)\r\n  File \"/home/vagrant/.ansible/tmp/ansible-tmp-1460189561.45-24282464722889/command\", line 1289, in _check_arguments\
r\n    for (k,v) in self.params.items():\r\nAttributeError: 'tuple' object has no attribute 'items'\r\n", "msg": "MODULE FAILURE", "parsed": false}
```

**Comment by sivel** [comment]
If I remember correctly, this usually means that you are using the v1 command module with v2 code base.  I see that you have defined a library at `/usr/share/ansible`

I would recommend removing that configuration from `/etc/ansible/ansible.cfg` and trying again.

**Comment by figure002** [comment]
@sivel Thanks for the quick reply, and you are right! I had two versions of Ansible installed; one from the Ubuntu repository (ansible 1.7.2), and one in a virtualenv (ansible 2.0.1.0). I decided to remove the one I installed from the repository (which also removes `/etc/ansible/ansible.cfg`), and that fixed it. The output from `ansible --version` now reads:

```
ansible 2.0.1.0
  config file = 
  configured module search path = Default w/o overrides
```
